working-groups/wg-talent-plan-courses.md

```diff
@@ -0,0 +1,29 @@
+# Talent Plan Courses Working Group
+
+## Goals
+
+THe Talent Plan Courses Working Group aims to develop a series of courses to help more peole enter the field of computer infrastructure system and the TiDB ecosystem.
+
+## Work and Meeting
+
+- Regular Working Group Meeting: 4pm on every Thursday.
+- Meeting Zoom: https://pingcap.zoom.com.cn/j/9681232794
+- [Meeting Notes](https://docs.google.com/document/d/1zDUoERn1MQYMR1PX0r0SuX8zlQIaQCjS9UhsbIBnn3g/edit)
```

Comment:
can the meeting notes open for the community?

Reply:
this is still under discussion. I will alter the content.